src/test/groovy/com/jvm_bloggers/entities/blog_post/BlogPostRepositorySpec.groovy

```diff
@@ -35,29 +35,29 @@ class BlogPostRepositorySpec extends SpringContextAwareSpecification {
 
     def "Should order latest posts by moderation and publication date"() {
         given:
-        Blog blog = aBlog("bookmarkId", "Top Blogger", "http://topblogger.pl/")
+        Blog blog = blogRepository.save(aBlog())
+        LocalDateTime baseTime = LocalDateTime.of(2016, 1, 1, 12, 00)
 
         List<BlogPost> blogPosts = [
-            aBlogPost(1, LocalDateTime.of(2016, 1, 1, 12, 00), REJECTED, blog),
-            aBlogPost(2, LocalDateTime.of(2016, 1, 4, 12, 00), REJECTED, blog),
-            aBlogPost(3, LocalDateTime.of(2016, 1, 2, 12, 00), APPROVED, blog),
-            aBlogPost(4, LocalDateTime.of(2016, 1, 5, 12, 00), APPROVED, blog),
-            aBlogPost(5, LocalDateTime.of(2016, 1, 3, 12, 00), NOT_MODERATED, blog),
-            aBlogPost(6, LocalDateTime.of(2016, 1, 6, 12, 00), NOT_MODERATED, blog)
+            aBlogPost(publishedDate: baseTime.plusDays(0), approved: REJECTED, blog: blog),
+            aBlogPost(publishedDate: baseTime.plusDays(1), approved: REJECTED, blog: blog),
+            aBlogPost(publishedDate: baseTime.plusDays(2), approved: APPROVED, blog: blog),
```

Comment:
For approved ones, we do not need to override the default value

Reply:
True, we don't. I thought the intention is more clear this way and consistent with the rest of them.